tests/end_to_end/misc1/irix-o2-validsyntax-out.c

```diff
@@ -1,3 +1,8 @@
+extern s32 D_410170;
+extern MIPS2C_UNK D_410178;
+extern MIPS2C_UNK func_00400140;
+extern MIPS2C_UNK func_00400158;
```

Comment:
I expect these will also change to real function declarations with the other PR?

Reply:
Yep! Exactly. 
(I wanted to keep the PRs separate because I think that makes it easier to review)

Reply:
Thanks for doing that, it helped!